src/gpodder/gtkui/main.py

```diff
@@ -1539,7 +1546,7 @@ def _for_each_task_set_status(self, tasks, status, force_start=False):
                         self.download_queue_manager.force_start_task(task)
                     else:
                         self.download_queue_manager.queue_task(task)
-                    self.enable_download_list_update()
+                    self.set_download_list_state(gPodderSyncUI.DL_ADDED_ONE)
```

Comment:
This PR introduces `gPodderSyncUI.DL_ADDED_ONE`, but this is not defined anywhere.
Steps to reproduce:

1. Download a file
2. Pause the download
3. Try to resume the download

```
Traceback (most recent call last):
  File ".../gpodder/src/gpodder/gtkui/main.py", line 1622, in <lambda>
    item.connect('activate', lambda item: self._for_each_task_set_status(tasks, status, force_start))
  File ".../gpodder/src/gpodder/gtkui/main.py", line 1567, in _for_each_task_set_status
    self.set_download_list_state(gPodderSyncUI.DL_ADDED_ONE)
AttributeError: type object 'gPodderSyncUI' has no attribute 'DL_ADDED_ONE'

```

Reply:
Filed #1137.